test/unit/behaviors.html

```diff
@@ -505,6 +593,37 @@
 
 });
 
+suite('templates from behaviors', function() {
+
+  test('template from registered callback', function() {
+    var el = fixture('from-registered');
+    assert.ok(el.shadowRoot.querySelector('#from-registered'));
+  });
+
+  test('template from base', function() {
+    var el = fixture('from-base');
+    assert.ok(el.shadowRoot.querySelector('#from-base'));
```

Comment:
The note inside this template says "should not be used" so it's confusing it's used here.

Since the behavior should set `_template` on the prototype, why isn't that preferred over the `dom-module`?

Reply:
Good catch. Made a change to `Polymer.Class` to ensure `_template` from behaviors take precedence over dom-module lookup for `is`, which both fixes a latent bug w.r.t. original Polymer 1.x behavior, and ensures that legacy elements that intend to get their template from a behavior are not subject to dom-module injection.